etc/apache2/sites-available/default-ssl.conf

```diff
@@ -0,0 +1,148 @@
+<IfModule mod_ssl.c>
```

Comment:
I think this file belongs in `extra`. Also, we probably don't need to specify the same name in the origin as in the destination, so maybe something like `extra/docker-apache.conf`? And then we can just rename it in the Dockerfile...